Microsoft.Toolkit.Uwp.UI/Extensions/ListViewBase/ListViewExtensions.cs

```diff
@@ -2,7 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
```

Comment:
Can we make this a partial class and create a `ListViewExtensions.Command.cs` file to hold the command specific implementation?

Reply:
Will do